docs/system-design.md

```diff
@@ -319,6 +319,14 @@ Mainnet reasonable range: 0 to 25e16 (0 to 25%)
 
 In order to restrict the system to organic patterns of behavior, we maintain two supply throttles, one for net issuance and one for net redemption. When a supply change occurs, a check is performed to ensure this does not move the supply more than an acceptable range over a period; a period is fixed to be an hour. The acceptable range (per throttle) is a function of the `amtRate` and `pctRate` variables. **It is the maximum of whichever variable provides the larger rate.**
 
+The recommended starting values for these parameters are as follows:
+|**Parameter**|**Value**|
+|-------------|---------|
+|issuanceThrottle.amtRate|250k|
```

Comment:
This assumes the RToken is worth $1, so I'd make clear the 250k is a $250k and they should really normalize this value by RToken worth.